jmx_prometheus_httpserver/src/main/java/io/prometheus/jmx/WebServer.java

```diff
@@ -18,6 +19,9 @@ public static void main(String[] args) throws Exception {
      Server server = new Server(port);
      ServletContextHandler context = new ServletContextHandler();
      context.setContextPath("/");
+     if (jc.gzipEnabled()) {
```

Comment:
If someone wants to turn it off we can consider it, but I'd avoid a knob we don't absolutely have to have.